rules-tests/Strict/Rector/Empty_/DisallowedEmptyRuleFixerRector/Fixture/empty_string_nullable.php.inc

```diff
@@ -20,7 +20,7 @@ final class EmptyStringNullable
 {
     public function run(string|null $value)
     {
-        return $value === null || $value === '';
+        return $value === null || $value === '' || $value === '0';
```

Comment:
`'0'` is also a empty string, therefore this test was wrong

see https://3v4l.org/mgSdd

Reply:
set `TREAT_AS_NON_EMPTY` to true seems make only compare null https://getrector.com/demo/204c8e52-11f0-492a-91a0-6b9f05c33531